### DISCUSSION TITLE
能不能改造下做个   hax   woiden的自动续期

### DISCUSSION THREAD

**Discussion by DifferentSparkler** [luolongfei/freenom#75] [Ideas]
https://hax.co.id/login
https://woiden.id/
大佬看看能不能改造下做这两个网站的续期

**Comment by luolongfei** [comment]
当然不能……hax 的续期其实我已经做了一个 py 版本了，但是那个站点经常变更验证方式，我暂时没时间对应，就还没公布，整个流程是走通了的

**Reply by DifferentSparkler** [reply]
谢谢大佬回复   期待早点发布
